.travis.yml

```diff
@@ -12,7 +12,6 @@ php:
 
 matrix:
   allow_failures:
-    - php: 7.1
```

Comment:
It is very discussable. In new version we have support from the box.

If you want to do it please change this lines as well

https://github.com/getsentry/sentry-php/blob/master/.scrutinizer.yml#L8 change to "7"
https://github.com/getsentry/sentry-php/blob/master/.travis.yml#L70 add new line

https://github.com/getsentry/sentry-php/pull/449#discussion_r113763595